src/examples/datepicker/DatepickerRange.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { DatepickerRange } from '@zendeskgarden/react-datepickers';
+import { Field, Label, Input, Message } from '@zendeskgarden/react-forms';
+import { compareAsc, addDays } from 'date-fns';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledCol = styled(Col)`
+  margin-bottom: ${p => p.theme.space.md};
+  max-width: 300px;
+`;
+
+const Example = () => {
```

Comment:
I've tweaked this example with a sandbox that I think provides an improved layout across devices along with support for `isCompact` styling: https://codesandbox.io/s/suspicious-johnson-34wks?file=/src/Example.tsx
...and more closely aligns with recommended grid [nesting](https://garden.zendesk.com/components/grid#nesting).

If we don't want the fields to remain at constant width with associated calendars, let's work on improving the expected responsive behavior (i.e. stacked fields).

Reply:
TY 👍